schema/verify/roles/idle-session-disconnector/grants.sql

```diff
@@ -0,0 +1,12 @@
+-- Verify seattleflu/id3c-customizations:roles/idle-session-disconnector/grants on pg
+
+begin;
+
+select 1/pg_catalog.has_database_privilege('idle-session-disconnector', :'DBNAME', 'connect')::int;
+select 1/pg_catalog.has_schema_privilege('idle-session-disconnector', 'public', 'usage')::int;
+select 1/pg_catalog.has_table_privilege('idle-session-disconnector', 'pg_stat_activity_nonsuperuser', 'select')::int;
+
+select 1/(not pg_catalog.has_table_privilege('idle-session-disconnector', 'pg_stat_activity', 'insert,update,delete'))::int;
+select 1/(not pg_catalog.has_table_privilege('idle-session-disconnector', 'pg_stat_activity_nonsuperuser', 'insert,update,delete'))::int;
+
```

Comment:
Would be good to test execute privs for the security-definer function, to verify that we got them right.

Reply:
Added a test to check that `idle-session-disconnector` has execute privileges but `public` does not.